modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -599,7 +599,7 @@ describe('TSS Ecdsa Utils:', async function () {
       const [userXShare, bitgoXShare] = await Promise.all([
         MPC.appendChallenge(userSigningKey.xShare, serializedEntChallenge),
```

Comment:
If `appendChallenge()` is no longer asynchronous, do we need to `await Promise.all()`?

Reply:
good catch, its no longer needed